manifests/charts/gateways/istio-ingress/templates/preconfigured.yaml

```diff
@@ -13,14 +13,15 @@ spec:
     istio: ingressgateway
   servers:
   - hosts:
-    - "*.global"
+    - "*.{{ .Values.global.multiCluster.globalDomainSuffix }}"
```

Comment:
Better trim space around the parameter `global.multiCluster.globalDomainSuffix`.